windows/src/developer/TIKE/oskbuilder/TouchLayoutDefinitions.pas

```diff
@@ -123,12 +123,12 @@   TJSONDef = record
     'rightalt',
     'rightctrl-rightalt',
     //'shift',
-    'shift-lctrl',
-    'shift-rctrl',
-    'shift-lalt',
-    'shift-ralt',
-    'shift-lctrl-lalt',
-    'shift-rctrl-ralt'
+    'leftctrl-shift',
+    'rightctrl-shift',
+    'leftalt-shift',
+    'rightalt-shift',
```

Comment:
Will this impact existing keyboards?

These layers are currently in use:

## shift-ralt
* basic_kbdarmph
* basic_kbdarmty
* basic_kbdcher
* basic_kbdinen

## shift-rightalt
* basic_kbdindev

Reply:
No, it's only used in the import from osk to touch layout. If we use those definitions elsewhere in the future, it's important that they match up, but even then existing keyboards will not be impacted.